server/store/sqlstore/store_test.go

```diff
@@ -1017,6 +1018,121 @@ func TestWhitelistIO(t *testing.T) {
 	assert.Nil(getErr)
 }
 
+func TestSetUserLastChatSentAt(t *testing.T) {
+	store, _ := setupTestStore(t)
+	assert := require.New(t)
+
+	mmUserID := model.NewId()
+	err := store.SetUserInfo(mmUserID, "ms-"+mmUserID, nil)
+	assert.Nil(err)
+
+	getLastChatSentAtForUser := func(mmUserID string) int64 {
+		t.Helper()
+		var lastChatSentAt int64
+		err = store.getQueryBuilder().
+			Select("LastChatSentAt").
+			From(usersTableName).
+			Where(sq.Eq{"mmuseriD": mmUserID}).
+			QueryRow().
+			Scan(&lastChatSentAt)
+		assert.Nil(err)
+		return lastChatSentAt
+	}
+
+	{
```

Comment:
I haven't seen this pattern of brackets -- can you elaborate on the intent? Should these be `t.Run` with a description instead of the comment?

Reply:
That was the sign of too long days and not enough will left to write `t.Run` tests 😅 i refactored it!